packages/rancher-vsphere/rancher-vsphere-cpi/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: local
-version: 100.1.0+up1.0.100
+version: 100.2.0+up1.1.100
```

Comment:
100.1.1+up1.1.100 should suffice right? Why do we need to bump the minor version